### ISSUE TITLE
Padding Controls

### ISSUE THREAD

**Issue by smabbasht** [cliffniff/media-controls#89]
Hey, I love this extension, Great work.
It would have been nicer if we could adjust padding instead of having solid left and right pads as attached in the screenshot.
Thanks

![image](https://github.com/cliffniff/media-controls/assets/79911677/573f7c71-ae73-4cdb-93a0-f2947cc7a1f8)

**Comment by ChrisLauinger77** [comment]
Have you tried changing the extension position in settings ?

**Comment by smabbasht** [comment]
Yes, I already tried that tab in settings along with any other tab there, it basically only allows to move it to center or right or only does it allow changing the index,
I was looking for something that would change the padding (the internal margin) not margin.

**Comment by ChrisLauinger77** [comment]
I am fine with Left, center , right setting.
There are 2 more Issue:  #52 & #68
